crates/aptos-rest-client/src/aptos.rs

```diff
@@ -24,3 +33,83 @@ impl Balance {
 pub struct AptosVersion {
     pub major: U64,
 }
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Account {
+    authentication_key: Vec<u8>,
+    sequence_number: u64,
+    self_address: AccountAddress,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+struct BlockMetadata {
+    /// Height of the current block
+    height: u64,
+    /// Time period between epochs.
+    epoch_internal: u64,
+    /// Handle where events with the time of new blocks are emitted
+    new_block_events: EventHandle,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct EventHandle {
+    /// Total number of events emitted to this event stream.
+    counter: u64,
+    /// A globally unique ID for this event stream.
+    guid: Guid,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Guid {
+    id: Id,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Id {
+    /// If creation_num is `i`, this is the `i+1`th GUID created by `addr`
+    creation_num: u64,
+    /// Address that created the GUID
+    addr: AccountAddress,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct NewBlockEvent {
+    epoch: u64,
+    round: u64,
+    height: u64,
+    previous_block_votes: Vec<bool>,
+    proposer: AccountAddress,
+    failed_proposer_indices: Vec<u64>,
+    /// On-chain time during the block at the given height
+    time_microseconds: u64,
+}
+
```

Comment:
We already have types and should use those rather than creating a secondary set. Where we don't, we should add to types.

Reply:
The purpose of this was to look at it separated between the internal types, and the external types in the event of a different language.  The base types we would need for each SDK.